### PR TITLE
feat(useDark): added initial value.

### LINKED ISSUE #1967: Enable default color scheme value in `useDark`
### Clear and concise description of the problem

By default, `useDark` uses `usePreferredDark` to set up the color scheme value by default. As a result, we cannot force  `light` as the default value.

### Suggested solution

This problem could be solved by implementing a `defaultValue` parameter in the `UseDarkOptions` payload, as follow:

```
import { useDark, useToggle } from '@vueuse/core'

const isDark = useDark({ defaultValue: "light" })
const toggleDark = useToggle(isDark)
```

### Alternative

_No response_

### Additional context

_No response_

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vueuse/vueuse/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vueuse/vueuse/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vueuse.org/guide).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### PR DESCRIPTION
**Pull request by chaii3**
closes #1967 
### Additional context

---

### What is the purpose of this pull request? 

- [ ] Bug fix
- [x] New Feature
- [ ] Documentation update
- [ ] Other

### Before submitting the PR, please make sure you do the following

- [x] Read the [Contributing Guidelines](https://github.com/vueuse/vueuse/blob/main/CONTRIBUTING.md).
- [x] Read the [Pull Request Guidelines](https://github.com/vueuse/vueuse/blob/main/packages/guidelines.md).
- [x] Check that there isn't already a PR that solves the problem the same way to avoid creating a duplicate.
- [x] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).
- [ ] Ideally, include relevant tests that fail without this PR but pass with it.

### CODE REVIEW

File: packages/core/useDark/index.ts [hunk 1]
```diff
@@ -55,6 +57,9 @@ export function useDark(options: UseDarkOptions = {}) {
     },
   })
 
+  if (options.initialValue)
+    mode.value = options.initialValue
```
Comment (antfu) [inline packages/core/useDark/index.ts:61]: If you are doing so, there will be no persistent at all

**Comment by orionna319** [comment]
Add an initial value for useColorMode?

File: packages/core/useDark/index.ts (hunk 1 above)
Comment (chaii3) [reply packages/core/useDark/index.ts:61]: Sorry I didn't understand

**Comment by tripflex** [comment]
```javascript
import { useColorMode, useToggle, useDark } from '@vueuse/core'

useColorMode({initialValue: 'dark'})

export const isDark = useDark()

export const toggleDark = useToggle(isDark)
```

**Pull request closed without merging**